src/legacy/core_plugins/status_page/public/lib/format_number.test.js

```diff
@@ -21,42 +21,42 @@ import formatNumber from './format_number';
 
 describe('format byte', () => {
   test('zero', () => {
-    expect(formatNumber(0, 'byte')).toEqual('0.00 B');
+    expect(formatNumber(0, 'byte')).toMatchInlineSnapshot(`"0.00 B"`);
   });
 
   test('mb', () => {
-    expect(formatNumber(181142512, 'byte')).toEqual('181.14 MB');
+    expect(formatNumber(181142512, 'byte')).toMatchInlineSnapshot(`"172.75 MB"`);
```

Comment:
FYI, these changes in the snapshots are expected because these tests were using normal 2 from npm, which doesn't have the fix we've applied in `@elastic/numeral` to divide bytes by 1024 rather than 1000. The snapshots now reflect the behavior that would be seen in the browser, since we shimmed `numeral` imports in the browser with `@elastic/numeral` ages ago.